docs/features/software-catalog/extending-the-model.md

```diff
@@ -447,6 +447,9 @@ want to have an isomorphic package that houses these types. Within the Backstage
 main repo the package naming pattern of `<plugin>-common` is used for isomorphic
 packages, and you may choose to adopt this pattern as well.
 
+You can generate an isomorphic plugin by running:`yarn new --select plugin-common`
+or you can run `yarn new` and then select "plugin-common" from the list of options
```

Comment:
```suggestion
You can generate an isomorphic plugin package by running:`yarn new --select plugin-common`
or you can run `yarn new` and then select "plugin-common" from the list of options
```

Technically not a plugin in itself, but a library

Reply:
Makes sense, I've added this 👍